tic-tac-toe/Rules.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+
+namespace tic_tac_toe
+{
+    public static class Rules
```

Comment:
I decided to use a static class here because I don't think it's necessary for `Rules` to have any state. Is it good practise to use static classes where possible? 

Maybe I should have a look if I can make any other classes static, too.

Reply:
I think this is a good use of a static class https://csharp.2000things.com/2013/02/20/784-when-note-to-use-a-static-class/